code/modules/awaymissions/maploader/reader.dm

```diff
@@ -321,7 +325,7 @@ var/global/dmm_suite/preloader/_preloader = null
 /dmm_suite/preloader/New(list/the_attributes, path)
 	.=..()
 	if(!the_attributes.len)
-		Del()
+		qdel(src)

```

Comment:
If you look at the Destroy proc, you'll see that it returns QDEL_HINT_HARDDEL_NOW. 
I didn't test this change (I don't even know how) but that qdel call will simply call Del() on the preloader anyway. No changed functionality; just different wording.